server/controllers/articles.js

```diff
@@ -214,6 +214,52 @@ class ArticleController {
       next(err);
     }
   }
+
+  /**
+  * Users can like an article
+  *
+  * @static
+  * @param {*} req
+  * @param {*} res
+  * @param {*} next
+  * @memberof ArticleController
+  * @return {json} Returns json object
+  */
+  static async like(req, res, next) {
+    try {
+      const me = req.user;
```

Comment:
Consider changing variable name ```me``` to something more descriptive to avoid confusion and explain better.
Consider using a phrase in camel casing.

Looks great to me otherwise.

Reply:
Thanks for your feedback but using me is descriptive enough. It simply refers to **the person** making the request.